app/views/onboardings/_task_card.html.erb

```diff
@@ -0,0 +1,43 @@
+<script>
+  function closeTaskCard() {
```

Comment:
i think it would be helpful to include a comment that points to `initializeOnboardingTaskCard` so that it's clear where the corresponding JS lives.

Reply:
Added!